sway/commands/exec_always.c

```diff
@@ -1,3 +1,4 @@
+#define _GNU_SOURCE
```

Comment:
no no no no no

Reply:
Aaaaaaaa.  I removed the call to `secure_getent(3)` but forgot to remove that.